README.md

```diff
@@ -8,10 +8,12 @@ Installation
 The go tool is the recommended method of installing goncurses. Issue the
 following command on the command line:
 
-$ go get github.com/rthornton128/goncurses
+```sh
+go get github.com/xaionaro-go/goncurses
```

Comment:
This line should be reverted `github.com/rthornton128/goncurses`. The rest of the MR looks fine.

Reply:
Fixed.